File: java/8.5/example3.java
import java.util.Scanner;

public class example3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Integer num = 10;
        System.out.println("Enter a number");
        Integer guess = Integer.parseInt(scanner.nextLine());
        if (num == guess) {
            System.out.println("That's correct");
        } else {
            System.out.println("Incorrect");
        }
    }
}

File: rust/src/8.5/example3.rs
fn main() {
    let num = 10;
    let mut guesss = String::new();
    println!("Enter a number");
    std::io::stdin().read_line(&mut guesss).unwrap();
    let guess = guesss.trim().parse::<i32>().unwrap();
    if num == guess {
        println!("Thats correct");
    } else {
        println!("Incorrect");
    }
}